src/features/unmarkedWsNoticeView/UnmarkedWsNoticeViewContainer.tsx

```diff
@@ -95,13 +98,34 @@ const UnmarkedWsNoticeViewContainer = () => {
     });
 
   return (
-    <UnmarkedWsNoticeView
-      customerProfile={customerProfile}
-      noticeDetails={noticeDetails}
-      winterStorageNotice={data.winterStorageNotice}
-      handleDeleteNotice={handleDeleteNotice}
-      handleLinkCustomer={handleLinkCustomer}
-    />
+    <>
+      <UnmarkedWsNoticeView
+        customerProfile={customerProfile}
+        noticeDetails={noticeDetails}
+        winterStorageNotice={data.winterStorageNotice}
+        handleDeleteNotice={handleDeleteNotice}
+        handleEditCustomer={() => setEditCustomer(true)}
+        handleLinkCustomer={handleLinkCustomer}
+      />
+
+      {customerProfile && (
+        <Modal isOpen={editCustomer} toggleModal={() => setEditCustomer(false)}>
+          <CustomerEditForm
+            customerId={customerProfile.customerId as string}
```

Comment:
Also here, would it be possible to avoid casting types?

Reply:
Yep, changed the util return type to assert that `customerId` is always there